default.ps1

```diff
@@ -78,7 +78,7 @@ task Nuget {
       @("RhinoDSL", "1.0.0"), `
       @("Common.Logging", "3.0.0"), `
       @("Common.Logging.Core", "3.0.0"), `
-      @("FileHelpers", "2.0.0.0") `
+      @("FileHelpers", "3.0.46-beta") `
```

Comment:
Looks like even with using the bracket notation that Nuget isn't allowing the build to happen. We're still getting the same Prerelease dependency error. 

@MarcosMeli, when do you plan on removing the -beta tag from 3.0.46?